### ISSUE TITLE
[MCP] Detail page edits

### ISSUE THREAD

**Issue by craigwheeler** [department-of-veterans-affairs/va.gov-team#33545]
## Issue Description

Verify values on the detail page are rendering correctly

- [x] Phone number in the blue status card should be 1-866-400-1238
- [x] ‘On this page’ should be h2 - let me know if this is part of the accessibility conversations we're having with Platform!
- [x] Margin between ‘On this page’ and ‘Download your statements’ should be 40px. This applies to all sections
- [x] The action link ‘Pay your copay bill online at pay.gov’ should open in a new tab so the Veteran can reference their account number while filling out the form 
- [x] Option 2: Pay by phone content needs to be updated. It’s using a placeholder phone number. ([UXPin for reference](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/142712286/simulate/sitemap?mode=i))
- [x] Option 4: Update content to ‘Visit [facility name] instead of ‘Visit your nearest VA medical center’
- [ ] Account number in downloaded statement: 7571001005127153 vs Account number on the page: 1005127153 

---
## Tasks
- [ ] all points in the bulleted list should be addressed

## Acceptance Criteria
- [ ] all values on the detail page should render correctly according to the outlined list

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by rileyorr** [comment]
# Staging review notes

## Overview page 

- [ ] Heads up: we’re currently chatting with Mikki about breadcrumb adjustments so stay tuned!
- [ ] Update subhead text to “Check the balance of VA health care and prescription charges from each of your facilities. Find out how to make payments or request financial help.”
    - [ ] Should also be using the intro text styling - refer to [the debt portal](https://staging.va.gov/manage-va-debt/) for styling 
- [ ] Contact information for your VA health care facilities: The second facility doesn’t have a contact number

## Detail page

- [ ] Update subhead text to use the intro text styling - refer to the debt portal 
- [ ] Phone number in the blue status card should be 1-866-400-1238
- [ ] ‘On this page’ should be h2 - let me know if this is part of the accessibility conversations we're having with Platform!
- [ ] Margin between ‘On this page’ and ‘Download your statements’ should be 40px 
   - [ ] This applies to all sections
- [ ] Account number in downloaded statement: 7571001005127153 vs Account number on the page: 1005127153 
- [ ] The action link ‘Pay your copay bill online at pay.gov’ should open in a new tab so the Veteran can reference their account number while filling out the form 
- [ ] Option 2: Pay by phone content needs to be updated. It’s using a placeholder phone number. ([UXPin for reference](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/142712286/simulate/sitemap?mode=i))
- [ ] Option 3: Update the content to ‘Your account number: [number]' & update the bullet point to ‘The facility you’d like to pay: [Facility name]'
- [ ] Option 4: Update content to ‘Visit [facility name] instead of ‘Visit your nearest VA medical center’

## Resolved copay balance detail page

- [ ] Facility name isn’t displaying
- [ ] Date isn’t displaying ‘Updated on Invalid date’ 
- [ ] The information alert ‘You don’t need to make aa payment at this time’ isn’t displaying 
- [ ] PDF statement has an invalid date 
- [ ] PDF statement cannot be downloaded
- [ ] Same accordion issues as noted above
- [ ] Account number isn’t being displayed in any of the accordions 
- [ ] Option 4: facility address isn’t displaying 
- [ ] Questions about your treatment or your charges - facility contact information isn’t populating

**Comment by craigwheeler** [comment]
tickets to address staging review feedback:

[33558: [MCP] Resolved copay balance detail page
](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/department-of-veterans-affairs/va.gov-team/33558)

**Comment by craigwheeler** [comment]
BE following up to ensure correct account number is being passed [33565: [MCP] Derive StatementId for FE](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/department-of-veterans-affairs/va.gov-team/33565) closing FE ticket
